Tests/scripts/infrastructure_tests/release_notes_generator_test.py

```diff
@@ -592,3 +593,30 @@ def test_construct_entities_block_indicator_types(self):
         rn = construct_entities_block(entities_data)
         assert '### Indicator Types' in rn
         assert '- **accountRep**' in rn
+
+    @pytest.mark.parametrize('pack_versions_dict, expected_results', [
+        ({'1.0.1': "#### Scripts\n***Breaking Change*** some change\n##### entity1\n- Fixed something\n"
+          "#### Integeration\n***Breaking Change*** some change",
+          '1.0.2': "#### Scripts\n***Breaking Changes*** some changes\n##### GetIncidentsByQuery\n"
+          "#### Integeration\n##### entity1\n- Fixed something"},
+         "#### Integeration\n***Breaking Change*** some change\n\n##### entity1\n- Fixed something\n\n#### Scripts\n"
+         "***Breaking Change*** some change\n***Breaking Changes*** some changes\n\n##### entity1\n- Fixed something")])
+    def test_merge_rns(self, pack_versions_dict, expected_results):
```

Comment:
Let's maintain the format and logic used in other tests regarding test data. See this as an example. 
https://github.com/demisto/content/blob/master/Tests/scripts/infrastructure_tests/release_notes_generator_test.py#L402.
Also ned to rename the function to a meaningful test name like `test_merge_rns_with_gerneral_announcment` or anything you think describes the use-case you are testing.